Applications/Core TLP.md

```diff
@@ -0,0 +1,70 @@
+## Introduction
+
+Currently the Node.js Foundation directly oversees the development of the core platform and associated working groups. The addition of Top Level Projects will predictably increase the workload on the TSC and so it makes sense to transfer these responsibilities to a Top Level Project concerned solely with the development of the platform.
+
+## History & Metrics
+
+Created by Ryan Dahl, now the largest programming ecosystem on the planet.
```

Comment:
It was a placeholder and hasn't been updated because we all know what node.js core is ;)